Makefile

```diff
@@ -25,14 +25,19 @@ prepare-for-rootfs: examples
 	@mkdir -p $(OUTPUT_DIR)
 	@mkdir -p $(OUTPUT_DIR)/ta
 	@mkdir -p $(OUTPUT_DIR)/ca
+	@mkdir -p $(OUTPUT_DIR)/plugins
 	@for example in $(EXAMPLE_LIST); do \
 		if [ -e $$example/host/optee_example_$$example ]; then \
 			cp -p $$example/host/optee_example_$$example $(OUTPUT_DIR)/ca/; \
 		fi; \
 		cp -pr $$example/ta/*.ta $(OUTPUT_DIR)/ta/; \
+		if [ $$example == plugins ]; then \
+			cp -p plugins/syslog/*.plugin $(OUTPUT_DIR)/plugins/; \
```

Comment:
Do we know for sure that `plugins/syslog/*.plugin` will match something here?

Reply:
Yes I think so. Do you have any particular scenario in mind where it might not be the case?

Also, there is possibility to use wildcard instead of `syslog` but I think that can come as an extension when we need to support multiple plugin examples.

Reply:
OK, fine then.